polaris-ui/src/app/common/presentation/layout/Layout.tsx

```diff
@@ -27,35 +28,46 @@ export const Layout: React.FC<LayoutProps> = ({ isWide, children }) => {
       (skipLinkSiblingRef.current as HTMLButtonElement).blur();
     }
   }, [location.pathname]);
+  const { globalNav } = useUserGroupsFeatureFlag();
```

Comment:
I've retained the existing `globalNav` feature flag property name to keep PR as small as possible.  Could be renamed to `globalHeader`.